test-case/check-sof-logger.sh

```diff
@@ -205,7 +205,12 @@ main()
 
             # Workaround for DMA trace bug
             # https://github.com/thesofproject/sof/issues/4333
-            if [ "$f" = data ]; then
+#            if [ "$f" = data ]; then
```

Comment:
Is this comment intentionally left in?  Also not sure I follow the reasoning for the `if false; then` line.  Guessing that's what the comments are referencing.

Reply:
This is just a test DRAFT, don't take it seriously.

The `if false` is a way to comment out 10 lines with a one-line change.